src/utils.js

```diff
@@ -138,3 +138,43 @@ return classes.map(c => {
   }
 })
 }
+
+// Uppon trigger downloads a blob of data to the browser with the given file name
```

Comment:
```suggestion
// Upon trigger downloads a blob of data to the browser with the given file name
```
fix cause I want to trigger a build